codenames-domain/src/main/java/com/codenames/mapper/RoomMapper.java

```diff
@@ -0,0 +1,32 @@
+package com.codenames.mapper;
+
+import com.codenames.models.game.Player;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+import com.codenames.dto.RoomDto;
+import com.codenames.models.room.Room;
+import org.mapstruct.factory.Mappers;
+
+
+@Mapper
+public interface RoomMapper {
+
+    RoomMapper INSTANCE = Mappers.getMapper(RoomMapper.class);
+
+    TeamMapper TEAM_MAPPER = Mappers.getMapper(TeamMapper.class);
+
+    PlayerMapper PLAYER_MAPPER = Mappers.getMapper(PlayerMapper.class);
+
+    WordsMapper WORDS_MAPPER = Mappers.getMapper(WordsMapper.class);
+
+
+    @Mapping(target = "blueTeam", expression = "java(TEAM_MAPPER.teamToTeamDto(room.getBlueTeam()))")
```

Comment:
I am sure some of these mappings can be done automatically or at least without specifying the whole `expression` param, just by `source` and `target`.